typing/includecore.ml

```diff
@@ -386,8 +408,8 @@ let report_type_mismatch first second decl env ppf err =
       pr "They have different arities."
   | Privacy err ->
       report_privacy_mismatch ppf err
-  | Kind ->
-      pr "Their kinds differ."
+  | Kind (kind1, kind2) ->
+      report_kind_mismatch ppf first second kind1 kind2
```

Comment:
It would be more consistent with other branches to have

```ocaml
| Kind err ->
  report_kind_mismatch first second ppf err
```

which requires slightly different definitions (probably a `kind_mismatch` type being defined as `type_kind * type_kind`.)

Reply:
Done!